README.md

```diff
@@ -87,6 +86,10 @@ Cheerify is a mobile application for iOS designed to uplift and inspire users th
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<div align="center">
+  <img src="https://github.com/Yashraj49/Cheerify/assets/88454884/afc46aee-9a6f-4f23-9b43-4eb22c2c9e4c" alt="Screenshot">
```

Comment:
upload this image to the repo if you can here. also nit: but a lot of the font sizes and punctuation in the design is inconsistent – specifically bottom left and bottom right text (both bold and footnote) should follow the same schema. The bottom middle affirmation should also not have either the first and second commas/exclamations.

Reply:
I can fix the issue with the size of the affirmation text in the middle, but I'm unsure about the text size of the content in the bottom left and right corners, as it appears to be using default settings.